### PR TITLE
CTC decoders

### PR DESCRIPTION
**Pull request by vsl9**
- Added CTC beam search decoder (module, package installation)
- Added Offline ASR notebook for greedy decoding, words splitting, decoding with N-gram re-scoring

### CODE REVIEW

**Comment by vsl9** [comment]
Reviewers, please replace `main` branch with `ctc-decoders` in the notebook for testing in Colab.

File: scripts/install_ctc_decoders.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+git clone https://github.com/NVIDIA/OpenSeq2Seq -b ctc-decoders
```
Comment (okuchaiev) [inline scripts/install_ctc_decoders.sh:1]: can you please change this to checkout a specific commit or tag?

**Review by okuchaiev — CHANGES_REQUESTED** [review]
thanks! looks good, just one minor comment

**Review by okuchaiev — APPROVED** [review]

**Comment by ShantanuNair** [comment]
Quick question - does this PR mean that beam-searching using a Language Model (kenLM) is now supported at least partially? Just trying to keep track of the state of things :) We're all eagerly keeping track of the progress you guys are making.

**Comment by ShantanuNair** [comment]
And apologies if this isn't the appropriate place for such questions (let me know) but it seems like the delay for the timestamps builds up more after each word, ending up with the timestamps not aligning with the right words at all. This could be because I'm running it on much longer (20-30minute) audio files instead of ~25s, but I figured that wouldn't be an issue since there's no RNN.

**Pull request merged**